include/log.h

```diff
@@ -99,6 +99,8 @@
 # endif
 #endif
 
+void pr_log_stacktrace(const char*moduleversion, const char*function);
```

Comment:
i modified in master the addition of module *m to auto generate mod/version string, file and lineno.

but fd is more complexe change, i'm not sure how to do properly and i think this type of debug backtrace are more needed in debug trace of proftpd than any other file...